x-pack/platform/plugins/shared/observability_solution/observability_ai_assistant/server/service/index.ts

```diff
@@ -98,6 +98,7 @@ export class ObservabilityAIAssistantService {
     const { asInternalUser } = coreStart.elasticsearch.client;
 
     const kbService = new KnowledgeBaseService({
+      request,
```

Comment:
I need the `request` to call the product doc API from the KB service. I tried working around it, but ultimately it seems that those "services" are request-scoped anyway so it felt fine to just do that,